frontend/src/use-cases/songs/components/song-form/SongForm.utils.jsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import {
+    DigitAutocompleteSelectMultiple,
+    DigitButton,
+    DigitDesign,
+    DigitLayout,
+    DigitMarkdown,
+    DigitTextArea,
+    DigitTextField,
+    useDigitCustomDialog,
+    useDigitFormField
+} from "@cthit/react-digit-components";
+import SongDetailContainer from "../song-detail-container";
+
+const definePreviewDialog = (songText, text) => ({
+    renderMain: () => (
+        <SongDetailContainer>
+            <DigitLayout.Column>
+                <DigitMarkdown markdownSource={songText} />
+            </DigitLayout.Column>
+        </SongDetailContainer>
+    ),
+    renderButtons: (confirm, cancel) => (
+        <DigitButton text={text.Close} raised onClick={confirm} />
+    )
+});
+
+export const SongFormFields = ({ text, tags }) => {
```

Comment:
Why not use `DigitEditData`?

Reply:
DigitEditData was wack. I had several problems which was solved when i switched to UseForm. Don't remember them specifically but i think it works pretty nice as of now.

Reply:
I think it mostly had to do with styling. DigitEditData was stupid hard to get the results I wanted while useForm was flexible and nice.

Reply:
Valid reason! I will link this to the rdc storybook as a usage example for `useDigitFormField`.